views/fields/select_single.tt

```diff
@@ -76,7 +76,7 @@
                   data-value="[% item.label_html %]"
               >
               <label class="radio-group__label" for="[% id %]_[% item.value %]">
-                [% INCLUDE fields/sub/filter.tt value=item.label_html filter=filter %]
+                [% INCLUDE fields/sub/filter.tt value=item.label_html filter="html" %]
```

Comment:
I've been looking a bit more at this, and I think it needs altering slightly. At the moment, the `select_single` expects `items` containing both `label_plain` and `label_html` keys (the latter being HTML entities encoded). I think adding the filter `html` as above will result in double-encoding when the widget is used for editing records.

I think you had the right idea originally, passing in the `filter` parameter. I suspect that `filter` is a reserved keyword though, so it would be worth trying again by renaming it (and also updating `edit.tt` which seems to have the wrong filter set already). Then it will also need `select_single.tt` updating to allow only only `label_plain` being provided in the case of `filter="html"` and automatically filling `label_html` in this case.

Hope that makes sense, if not can discuss next time I'm in!